src/scripts/controllers/addClientThirdPartyController.ts

```diff
@@ -29,6 +30,68 @@ export async function getAddClientThirdParty(req: Request, res: Response, next:
   });
 }
 
+/**
+ * Extract third party form fields from request body
+ * @param {unknown} body - Request body
+ * @returns {Record<string, unknown>} Extracted form fields
+ */
+function extractThirdPartyFormFields(body: unknown): Record<string, unknown> {
```

Comment:
I'm confused - this is the "remove third party" feature, why do we need to rework the "add third party" feature?